content/docs/faq.md

```diff
@@ -11,13 +11,20 @@ aliases: [/faq]
 
 Thanks for your interest in Packit Service! In order to start using the
 service, your repository or namespace needs to be allowed.
-For each installation we create a new issue in our
-[allowlist tracker](https://github.com/packit/notifications/issues/)
-to resolve this.
 Just be aware that we are now onboarding Fedora contributors who have
 a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System) account.
-So provide us enough information (your FAS account) in the issue.
-Until that, you will get a neutral status with `Namespace is not allowed!` message on your commits.
+
+We require our new users to have `GitHub Username` field set in the FAS account so that we can automatically
+check for the match between the `GitHub Username` field in the  
+FAS account and the Github account that triggers the verification. 
+For each installation, we create a new issue in our
+[allowlist tracker](https://github.com/packit/notifications/issues/)
+to resolve this, so you can just provide the FAS account in a comment
+as `/packit verify-fas my-fas-username` in the issue we create 
+for your installation. This will trigger the automatic check.
+Until that, you will get a neutral status with `Namespace is not allowed!` 
```

Comment:
Could this message be something like: `github.com/<namespace> needs to be approved before building.` And maybe link to this documentation.

Reply:
This was already adjusted in the past, but yes, this sounds more straightforward. I can open an issue in packit-service for this.

Reply:
Thanks!